<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>650</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>显示时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加载界面</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="webEngineView" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1201</width>
      <height>531</height>
     </rect>
    </property>
    <property name="url" stdset="0">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
